src/fastga/models/aerodynamics/components/wing/compute_cl_wing.py

```diff
@@ -0,0 +1,83 @@
+"""Estimate wing lift coefficient."""
+#  This file is part of FAST-OAD_CS23 : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2025  ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import openmdao.api as om
+
+
+class ComputeWingLiftCoefficient(om.ExplicitComponent):
+    """
+    Computation of the wing lift coefficient.
+    """
+
+    # pylint: disable=missing-function-docstring
+    # Overriding OpenMDAO initialize
+    def initialize(self):
+        self.options.declare("low_speed_aero", default=False, types=bool)
+
+    # pylint: disable=missing-function-docstring
+    # Overriding OpenMDAO setup
+    def setup(self):
+        ls_tag = "low_speed" if self.options["low_speed_aero"] else "cruise"
+        aoa = np.deg2rad(5.0) if self.options["low_speed_aero"] else np.deg2rad(1.0)
+
+        self.add_input("data:aerodynamics:wing:" + ls_tag + ":CL0_clean", val=np.nan)
+        self.add_input(
+            "data:aerodynamics:wing:" + ls_tag + ":CL_alpha", val=np.nan, units="rad**-1"
+        )
+        self.add_input(
+            "settings:aerodynamics:reference_flight_conditions:" + ls_tag + ":AOA",
+            units="rad",
+            val=aoa,
+        )
+
+        self.add_output("data:aerodynamics:wing:" + ls_tag + ":CL_wing", val=0.7, units="rad**-1")
```

Comment:
A lift coefficient is dimensionless or `unitless` not `rad**-1`
Also, I'd like to consider whether or not we really want to promote that output. It's only going to be used locally, no ? In which case it can just be name `CL_wing` and not be promoted. Alternatively, if it used by more than one components which computes derivative, it should be a component of its own and should have a clear name and desc mentioning that it will only be used for aerodunamic derivative